### DISCUSSION TITLE
Feasibility of cross-platform MSI

### DISCUSSION THREAD

**Discussion by kamilcieciura-okta** [wixtoolset/issues#7580] [Questions]
Hello,

I need an installer that can work for both 32-bit and 64-bit OS. Currently I build 2 separate MSIs and bundle them together in a bootstrapper EXE. The problem is that 90% of the files are common between the two platforms and my installer is almost double of what it can be. I tried making a 32-bit MSI that would install common files, but it can't access the 64-bit OS folders like Program Files or System32. I tried using a merge module for common files, but the 2 separate MSIs that reference the merge module end up being the same size as without using the merge module. Would making the 2 MSIs share an external cab file work? Is there a standard way to solve this problem?

I’m using WiX Toolset 3.11.

Thank you.

**Comment by BMurri** [comment]
A shared CAB file is possible, but it's been years since I've done it, so all I have time to do for now is to encourage you. We used to call the relevant feature "cab reuse".
